docs/contributing/development.md

```diff
@@ -159,3 +159,29 @@ the latest changes from the master branch.
 ❯ git remote add upstream https://github.com/neutrinojs/neutrino.git
 ❯ git pull upstream master
 ```
+
+## Releasing a new version
+
+1. Decide whether the new version should be a major/minor/patch/prerelease version bump.
+2. From the root of the Neutrino repository, run: `git checkout -b version-bump master`
+3. Then run `yarn release:prepare` and pick the desired new version.
+4. Check the changes in the working directory and adjust if necessary.
+
+    If bumping to a new major version, or incrementing the pre-release version,
+    you will want to manually increase the presets' version of `neutrino` in
+    `peerDependencies`.
```

Comment:
Maybe we could also add a note here on how to make this easier via `sed`? This is the command I use on macOS:

```sh
sed -i '' -- 's/"neutrino": "^9.0.0-0"/"neutrino": "^9.0.0-rc.0"/g' packages/*/package.json(D.)
```

Reply:
Yeah good idea. The example sed command in your comment didn't work for me, however this did:
`sed -i 's/"neutrino": "^9.0.0-0"/"neutrino": "^9.0.0-rc.0"/g' packages/*/package.json`

It might be worth testing that on your machine to check the OS X `sed` (which iirc is a little quirky) handles that form ok (or whether it needs an additional `-e` before the script command).